sdk/storage/tests-install-azurite.yml

```diff
@@ -0,0 +1,11 @@
+steps:
+  - task: Npm@1
+    inputs:
+      command: custom
+      customCommand: install --prefix $(Agent.TempDirectory)/azurite azurite
+    displayName: Install Azurite
+  - task: PowerShell@2
+    displayName: Add Azurite location as environment variable
+    inputs:
+      targetType: 'inline'
+      script: Write-Host "##vso[task.setvariable variable=Azure.Azurite.Location;]$(Agent.TempDirectory)/azurite"
```

Comment:
Does this end up getting correctly setup in the environment for the test runs?

Reply:
Yes. I have verified that tests depending on that variable run and pass in the CI build for this PR.